atom/browser/native_window.cc

```diff
@@ -805,6 +810,25 @@ void NativeWindow::ScheduleUnresponsiveEvent(int ms) {
       base::TimeDelta::FromMilliseconds(ms));
 }
 
+void NativeWindow::SetHtmlApiFullscreen(bool enter_fullscreen) {
```

Comment:
The definition of member functions should be placed in the same order with their declarations.